packages/nx/src/command-line/connect/connect-to-nx-cloud.ts

```diff
@@ -48,7 +48,8 @@ export async function connectToNxCloudIfExplicitlyAsked(
 
 export async function connectToNxCloudCommand(): Promise<boolean> {
   const nxJson = readNxJson();
-  if (isNxCloudUsed(nxJson)) {
+  const [isCloudUsed, _] = isNxCloudUsed(nxJson);
```

Comment:
The second parameter is not used so there is no need to define it.

```suggestion
      const [isCloudUsed] = isNxCloudUsed(originalNxJson);
```